code/datums/mind.dm

```diff
@@ -1031,12 +1031,12 @@
 			if("clear")
 				remove_antag_datum(/datum/antagonist/changeling)
 				special_role = null
-				to_chat(current, "<span class='userdanger'>You grow weak and lose your powers! You are no longer a changeling and are stuck in your current form!</span>")
+				to_chat(current, "<span class='userdanger'>We grow weak and lose our powers! You are no longer a changeling and are stuck in your current form!</span>")
 				message_admins("[key_name_admin(usr)] has de-changeling'ed [current].")
 				log_admin("[key_name(usr)] has de-changeling'ed [current].")
 			if("changeling")
 				var/datum/antagonist/changeling/C = make_Changling()
-				to_chat(current, "<span class='boldannounce'>Your powers are awoken. A flash of memory returns to us...we are [C.changelingID], a changeling!</span>")

```

Comment:
This removes important lore regarding Egypt, specially regarding the racial distribution of the people in royalty.

Reply:
Thanks for reviewing my code!